ONSdigital/sdx-transform-cs
# Add formtype 0070 to stocks

## What? and Why?
Needed to add new formtype 0070 for QSI (Stocks)

## How to test
Submit a version using console (get the JSON for it from either the pck unit tests or modify one already in console).  Verify the pck and images look reasonable.

## Checklist
